setup.cfg

```diff
@@ -60,6 +60,7 @@ install_requires =
     requests >= 2.22.0
     GitPython >= 3.0.8
     dmgbuild >= 1.3.3; sys_platform == "darwin"
+    psutil ~= 5.9.0
```

Comment:
PyCharm automatically added this.

As for the version, I think it is safe to require a recent version of `psutil` since I think the expectation is for users to use a dedicated virtual environment....although, I wouldn't be surprised if users installed briefcase in to their app's virtual environment.

As for the `~=`, I usually prefer it over `>=` but wasn't sure if you might have a reason to avoid it.